mantidproject/mantid
# Indirect - Add Workflow diagram to SimulatedDensityOfStates algorithm

Added a basic workflow diagram to SimulatedDensityofStates Documentation page.

**To test:**
- Build the documentation and ensure the workflow diagram is present and give an overview of the algorithms operation

Fixes #15631.
This issue should also close umbrella issue #14975

[Release Notes](https://github.com/mantidproject/mantid/blob/15631_Workflow_for_SDOS/docs/source/release/v3.7.0/indirect_inelastic.rst#improvements)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
